spec/features/review_card_spec.rb

```diff
@@ -1,8 +1,23 @@
 require 'rails_helper'
 
-feature 'Review Card', %q{'It should
-  be the ability to
-  check the knowledge Card'} do
+feature 'Review Card', 'It should be the ability to check the knowledge Card' do
+
+  before do
+    Card.create(original_text: 'hello', translated_text: 'Привет', review_date: '01.05.2015')
+    visit root_path
+  end
+
+  scenario 'enter the correct translation' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.